<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="25" started-at="2019-08-11T15:15:01Z" finished-at="2019-08-11T15:15:01Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="25" started-at="2019-08-11T15:15:01Z" finished-at="2019-08-11T15:15:01Z">
      <class name="TestPractice.Guru99Bank.TestBase.ExcelUtility">
        <test-method status="FAIL" signature="readData(java.lang.String)[pri:0, instance:TestPractice.Guru99Bank.TestBase.ExcelUtility@2d209079]" name="readData" duration-ms="0" started-at="2019-08-11T20:45:01Z" finished-at="2019-08-11T20:45:01Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method readData requires 1 parameters but 0 were supplied in the @Test annotation.]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method readData requires 1 parameters but 0 were supplied in the @Test annotation.
	at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:198)
	at org.testng.internal.Parameters.createParameters(Parameters.java:134)
	at org.testng.internal.Parameters.createParameters(Parameters.java:373)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:450)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- readData -->
      </class> <!-- TestPractice.Guru99Bank.TestBase.ExcelUtility -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
